blemmy/feed/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailcore.fields import RichTextField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel, InlinePanel, MultiFieldPanel, FieldRowPanel
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from wagtail.wagtailsearch import index
+# API model
+from wagtail.api import APIField
+
+# Create your models here.
+
```

Comment:
You might want to expand on the default Wagtail home page model using something like https://github.com/datalets/public-health-ch/blob/master/publichealth/home/models/models.py#L161